src/luarocks/cmd/test.lua

```diff
@@ -23,15 +23,16 @@ to separate LuaRocks arguments from test suite arguments.]],
       :args("?")
    cmd:argument("args", "Test suite arguments.")
       :args("*")
-
+   cmd:flag("--prepare", "Install all dependencies, including the test ones")
```

Comment:
Perhaps a clearer description of the behavior, like this?

```suggestion
   cmd:flag("--prepare", "Only install dependencies needed for testing only, but do not run the test")
```